src/www/app/outline_server_repository/index.ts

```diff
@@ -188,8 +188,15 @@ export class OutlineServerRepository implements ServerRepository {
     }
     for (const serverId of Object.keys(configById)) {
       const config = configById[serverId];
+
+      const serverConfig = new OutlineServerConfig(config.tag.data, config);
```

Comment:
This will break, we need to figure out how to deal with `ServersStorageV0` - we're sure we can't remove by now?